### ISSUE TITLE
Desktop UI updates

### ISSUE THREAD

**Issue by starryeyez024** [htothenan1/min-waste#6]
1. When adding new groceries from the list, I found myself tapping the name of an item like avocados, then hitting the "Add" button. Maybe the adding of custom items would be less confusing if it appeared over the top of the users personal list of items instead? 
2. Perhaps the food item containers could be a little wider to show more of the name?
3. I don't think Tailwind supports this out of the box, but on desktop, you could add some custom CSS that takes into account the height of the screen, and then makes the input fields that contain the food items much taller. 
```
@media screen and (min-height: 800px) { ... }
```

This is not a great mockup, just me messing around in web inspector, but it conveys the idea :) 
  
<img width="910" alt="image" src="https://github.com/htothenan1/min-waste/assets/456863/b87b7ad9-f2af-4bb5-a7fc-1857592ef1b5">

**Comment by htothenan1** [comment]
umm, love it

**Comment by htothenan1** [comment]
made some impulsive commits cause we have no users.

<img width="1224" alt="image" src="https://github.com/htothenan1/min-waste/assets/50812996/81e85127-32c0-4e98-a573-70abba0d8be2">

**Comment by starryeyez024** [comment]
LOL looks great! 👏🏻 You may want to left align the text in the groceries list too, just for consistency. I think there are accessibility studies that show it's easier for your eye to traverse through a list if they are left aligned (at least for LTR languages). 
Then that begs the question, "should the titles be centered or left-aligned too?" That's more of a stylistic choice though. :)

**Comment by htothenan1** [comment]
is it? is it jarring to your eye to have the titles centered, but list items left-aligned?

**Comment by starryeyez024** [comment]
I would probably left-align them all just out of personal preference, but I think it's less important because you only need to read the titles once. Whereas the list items you might be scanning up and down and so those should be left-aligned for ease of reading. Up to you!

**Comment by htothenan1** [comment]
<img width="996" alt="image" src="https://github.com/htothenan1/min-waste/assets/50812996/c8696ba4-e4b7-412f-9668-b441dfd22d3e">

**Comment by starryeyez024** [comment]
Hooray, it looks so good!

**Comment by htothenan1** [comment]
@starryeyez024  clear fridge button, in Account (for now)

<img width="552" alt="image" src="https://github.com/htothenan1/min-waste/assets/50812996/d4275762-7dd4-4a6f-b4c8-52f749bb4cdd">

**Comment by starryeyez024** [comment]
Awesome, thank you! Is it on the account settings page just from an ease-of-development perspective for now?

**Comment by htothenan1** [comment]
Yeah, I don't really know where to put it on the Kitchen page, or even if it belongs there (it probably should be on the same page as the list). Also, I figured it wouldn't hinder the UX too much if you had to go to Account before flushing your items. 

Also, the Kitchen page is already clogged with that super complicated conditional overlay that I want to clean up before adding new actions.

**Comment by htothenan1** [comment]
@starryeyez024 FYI as a sidenote, I want to start decoupling tailwind (I have a LOT of stupid css classes floating around) from this project, and revert everything back to regular CSS. I am also in the process of cleaning up my dependencies, getting rid of headlessui and radix. I feel like making these changes will allow this project to scale a bit better, and also make it easier to contribute to. Would love your thoughts.
